src/js/translations/en-US.json

```diff
@@ -25,13 +25,34 @@
   "SERVICE_ACTIONS.SCALE_BY": "Scale By",
   "SERVICE_ACTIONS.SUSPEND": "Suspend",
 
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_1": "Copy and paste this command into the CLI",
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_2": "Then, run the",
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_3": "for each service to clean up reserved resources",
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_4": "Then",
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_5": "the group",
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_6": "Copy the command below into the DC/OS CLI",
```

Comment:
For anyone wondering why template strings aren't used, it's because we can't use the `FormattedMessage` component here because this text gets rendered inside a Modal, and the Modal is rendered in a separate context than the rest of the app. Therefore we need to use the `formatMessage` function provided by the `injectIntl` HOC, which doesn't allow React components to be rendered in a template string.